change/@microsoft-teams-js-b23ef57f-9184-4659-93c8-48d862788dd3.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Made `app`, `appInitialization`, and `settings` treeshakable as well as added dts rollup plugin",
```

small nit and feel free to ignore it if you disagree. 
```suggestion
  "comment": "Made the `app`, `appInitialization`, and `settings` files treeshakable and added dts rollup plugin",
```

Additionally, does the dts rollup plugin only impact internal teams? If so we can probably just remove the dts part from this.

---

Updated!